modules/ebi-metagenomics/dram/distill/main.nf

```diff
@@ -0,0 +1,54 @@
+process DRAM_DISTILL {
+    tag "$meta.id"
+    label 'process_high'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/dram:1.3.5--pyhdfd78af_0':
+        'quay.io/biocontainers/dram:1.3.5--pyhdfd78af_0' }"
+
+    containerOptions="--bind ${moduleDir}/tests/fixtures/dram_distill_dbs/:/data/ --bind ${moduleDir}/tests/fixtures/dram_distill_dbs/CONFIG:/usr/local/lib/python3.10/site-packages/mag_annotator/CONFIG"
+
+    input:
+    tuple val(meta), path(dram_input)
```

Comment:
I would maybe rename dram_input to something more meaningful depending on its format? is it a tsv, fasta, etc? one example could be `tsv_input`